### ISSUE TITLE
Did you ever figure this out?

### ISSUE THREAD

**Issue by tmarkovski** [k0nserv/rust-swift-test#1]
I'm running into the same exact issue and I was wondering if you were able to find a solution to this. I wonder if adding module maps somehow makes it work, but I wasn't able to make progress.

**Comment by k0nserv** [comment]
Yes it did and in the same way you discovered in #2. Sorry for not getting back faster on this I missed the notification

**Comment by k0nserv** [comment]
I should archive this repo
